src/main/java/org/jarling/v2/api/AccountHolderResource.java

```diff
@@ -14,4 +16,9 @@
      * Get name of the account holder
      */
     String getAccountHolderName() throws StarlingBankRequestException;
+
+    /**
+     * Get profile image of the account holder encoded as a Base64 string
```

Comment:
Soo, is it a Base64 string(like the comment says), or a byte array(as the type and name suggest)? I imagine the latter.

Reply:
It is a byte array! Sorry, had changed the code but not this comment.